session/bootstrap.go

```diff
@@ -475,6 +477,10 @@ func upgrade(s Session) {
 		upgradeToVer28(s)
 	}
 
+	if ver < version29 {
+		upgradeToVer29(s)
```

Comment:
In `upgradeToVer28`, the `CreateBindInfoTable` already contains the `primary_index`. So, I guess the condition should be `if ver == version28`.